### DISCUSSION TITLE
Access keys with semi-colon in name

### DISCUSSION THREAD

**Discussion by zzph** [mikefarah/yq#1349] [Q&A]
Hi there,

This seems like a super handy library. I'm wondering, how I can access keys with a semi colon in it?

I tried `yq '..vars:foo:bar' < .my.yml`  but it tells me `invalid input text`.

What is the correct way to do this?

## My `.yml` file:
```
.vars:foo:bar:
  baz:
    DOMAIN: 'test.com'
```

**Comment by mikefarah** [comment]
Hi - take a look at the special character syntax here: https://mikefarah.gitbook.io/yq/operators/traverse-read#special-characters

**Comment by ringerc** [comment]
`.[".vars:foo:bar"]`

e.g.

```
$ yq '.[".vars:foo:bar"]' <<'__END__'
.vars:foo:bar:
  baz:
    DOMAIN: 'test.com'
__END__
baz:
  DOMAIN: 'test.com'
```
